### ISSUE TITLE
Open-API Processing - Linked tables

### ISSUE THREAD

**Issue by app-generator** [app-generator/sample-django-tastypie#6]
The tools should be able to process nested tables. Something like:
- product (1 -> M) 
  - same product belongs to multiple sales 
- sales (1-1 relationship)
 - a sale is linked to a single product 

> Here is the input file: [oa_product_sales.json](https://github.com/app-generator/sample-django-tastypie/blob/main/inputs/oa_product_sales.json)

---

## Desired Flow:

- Input file is parsed via CLI and models are generated
- CFG Should be updated (if required)
- API via TastyPIE should be able to manage both tables 
- CRUD methods: 
  - READ (public  request)
  - CREATE (restricted )
  - UPDATE (restricted)
  - DELETE (restricted) 

Constraints & Use-cases 
- Guests users (no auth required)
  - GET All products
  - GET product by ID
  - GET ALL Sales
  - GET Sale by ID 
- Authenticated Users:
  - Create Products
  - Create Sales using existing Product 
    - If products is not defined, sales creation fails 

basically, the usual API code for a simple, secured API. 

The Open-Api File can be generated using this tool:
https://appseed.us/generator/openapi/

For the mentioned INPUT, here is the visual representation: 

![image](https://user-images.githubusercontent.com/51070104/214537439-24ff64e9-6bc3-471f-8408-6da43226f4b5.png)

**Comment by app-generator** [comment]
Hello @MySecondLanguage 

Thank you for the latest commit. 
Please document here the flow (all steps explained).

Ty!
